CRM/Core/BAO/CMSUser.php

```diff
@@ -198,10 +198,16 @@ public static function formRule($fields, $files, $form) {
         $params = [
           'name' => $fields['cms_name'],
           'mail' => $fields[$emailName],
+          'pass' => $fields['cms_pass'],
```

Comment:
This gives an undefined key error if the cms doesn't allow the user-entered password. But I'd be ok to fix that in the other PR after. Since either way you'll need to fix merge conflicts in one of the PRs, I'll let you decide which one you'd rather do first.

Reply:
@demeritcowboy okay I'll make a note of this and fix this in the other PR.